stratosphere/ncm/source/impl/lr_redirection.cpp

```diff
@@ -18,54 +18,75 @@
 
 namespace sts::lr::impl {
 
+    class LocationRedirection  : public util::IntrusiveListBaseNode<LocationRedirection> {
```

Comment:
Move this inside an anonymous namespace in here, so that it doesn't get exposed to other translation units.

Reply:
It seem this is still not inside an anonymous namespace?